compiler/rustc_codegen_cranelift/Cargo.toml

```diff
@@ -16,7 +16,7 @@ cranelift-jit = { version = "0.98", optional = true }
 cranelift-object = { version = "0.98" }
 target-lexicon = "0.12.0"
 gimli = { version = "0.27.2", default-features = false, features = ["write"]}
-object = { version = "0.30.3", default-features = false, features = ["std", "read_core", "write", "archive", "coff", "elf", "macho", "pe"] }
+object = { version = "0.30.4", default-features = false, features = ["std", "read_core", "write", "archive", "coff", "elf", "macho", "pe"] }
```

Comment:
Could you please back out this change too? It will conflict with the next cranelift update.

Reply:
Ok